### ISSUE TITLE
Do manual checks on high volume documents

### ISSUE THREAD

**Issue by mcharg** [DOI-ONRR/onrr.gov-site#1274]
So we can see the difference between what comes out of manual versus automatic checks.

**PDF**
PDF form: https://www.onrr.gov/reportpay/PDFDocs/2014.pdf
PDF contacts: https://www.onrr.gov/ReportPay/PDFDocs/royassign.pdf
PDF handbook: https://www.onrr.gov/ReportPay/Handbooks/pdfdocs/RRM-Printable.Minerals.Revenue.Handbook.pdf

**Excel**
Excel contacts: https://www.onrr.gov/ReportPay/PDFDocs/ALwState.xlsx
Excel reference list: https://www.onrr.gov/reportpay/PDFDocs/CRLNumber.xlsx

**PowerPoint**
PPT: https://www.onrr.gov/reportpay/PDFDocs/Industry.WebCenter.Portal.User.Guide.pptx

**Comment by cthomasONRR** [comment]
A couple of thoughts...

It doesn't make sense to do manual checks on the PDFs. We should be doing those on the source Microsoft documents. Setting up and tracking reading order is much easier in Microsoft. Color contrast isn't much of an issue with these docs because they are all black+white.

For the excels, my current understanding is that as long a there aren't merged cells, it's mostly good to go. Manual color contrast checks are needed for both of these excels, and can be easily remediated if contrast if poor by choosing a different fill color.

For the PowePoint, manually checking reading order is easy. You click the down carrot on the "Check Accessibility" button, and can open the reading order pane. You can then click through and rearrange the reading order in that pane. The same process goes for checking the Alt Text. The biggest issue I see is that manually checking all color contrasts will get very time-consuming on a 50+ slide presentation.

**Comment by jkwonatx** [comment]
Revisiting this issue again. Will come up with a plan to resolve it.

**Comment by jkwonatx** [comment]
I'll revisit today and comment.

**Comment by jkwonatx** [comment]
@mcharg @cthomasONRR Now that I have done a manual check on a word doc, I feel better about tackling this -- happy to pick this back up whenever

**Comment by cthomasONRR** [comment]
After some thought, maybe this should be a checklist we use and when something fails, we send back to document owner? That way, we are not spending hours manually checking documents.

**Comment by eelzi-ONRR** [comment]
@cthomasONRR I was looking for the document issue you just worked on and found this one. I like your idea of making a manual checklist for document owners. Can I change this issue to reflect that and you and I can work on it eventually? I don't think we need to do what this issue originally asks since you just did that

**Comment by cthomasONRR** [comment]
@eelzi-ONRR Agreed! Since I already manually tested the top 10 downloads, this issue can either be closed or like you mentioned- repurposed to make a manual checklist for doc owners.
